src/Shared/SharedSecurityDetails.ts

```diff
@@ -150,6 +150,11 @@ export const getBreakdown = (
     table.setColumn(assetStr);
     table.setColumn(notClassifiedStr);
 
+    // Early return if no breakdown.
+    if (!breakdown || breakdown.length === 0) {
+        return;
+    }
```

Comment:
We might need to do that in all of the converters so let's keep that in mind while implementing it

Reply:
In all converters planned for refactoring (XRay)? Or a general rule? Perhaps a more detailed comment would be better?

Reply:
More like a general rule, to pay attention to empty returns in all other converters (that do not use `getBreakdowns` method)

Reply:
Okay, the question is then: when should it happen? Before or after setting up the columns? Currently we check it twice..

![image](https://github.com/user-attachments/assets/1d7361df-69dc-4a74-b3f4-cbd48f956b37)


Should we introduce another guesrd, like `isValid()` in parent abstract class? 🤔 Or sth similar